### PR TITLE
Update guided-scenarios-cloud-managed-pc.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated prerequisites

per issue#https://github.com/MicrosoftDocs/memdocs/issues/3132

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/fundamentals/guided-scenarios-cloud-managed-pc.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ If you want to evaluate a cloud-managed modern desktop in your own organization,
 ## Prerequisites
 
 - [Set the MDM authority to Intune](../fundamentals/mdm-authority-set.md#set-mdm-authority-to-intune) - The mobile device management (MDM) authority setting determines how you manage your devices. As an IT admin, you must set an MDM authority before users can enroll devices for management.
-- M365 E3 minimum (or M365 E5 for best security)
+- M365 Business premiuum or M365 E3  or M365 E5 for best security.
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/guided-scenarios-cloud-managed-pc.md:34]:
```suggestion
- M365 Business Premium, Microsoft 365 E.3,  or Microsoft 365 E5 for best security.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: memdocs/intune/fundamentals/guided-scenarios-cloud-managed-pc.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ If you want to evaluate a cloud-managed modern desktop in your own organization,
 ## Prerequisites
 
 - [Set the MDM authority to Intune](../fundamentals/mdm-authority-set.md#set-mdm-authority-to-intune) - The mobile device management (MDM) authority setting determines how you manage your devices. As an IT admin, you must set an MDM authority before users can enroll devices for management.
-- M365 E3 minimum (or M365 E5 for best security)
+- M365 Business Premium, Microsoft 365 E.3,  or Microsoft 365 E5 for best security
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/guided-scenarios-cloud-managed-pc.md:34]:
```suggestion
- Microsoft 365 Business Premium, Microsoft 365 E.3,  or Microsoft 365 E5 for best security
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please review and signoff this PR. Thanks

**Comment by Erikre** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**